internal/impl/generate.go

```diff
@@ -61,7 +61,16 @@ func (d *Devbox) generateShellFiles() error {
 	}
 
 	for _, pkg := range d.packages() {
-		if err := d.pluginManager.CreateFilesAndShowReadme(d.writer, pkg, d.projectDir); err != nil {
+		if err := d.pluginManager.Create(d.writer, pkg, d.projectDir); err != nil {
+			return err
+		}
+	}
+
+	for _, includes := range d.cfg.Include {
```

Comment:
nit: shouldn't this be singular `include`?

Reply:
actually, in addition, I'm being thrown off by `d.cfg.Include` being singular....should that be plural? 

Relatedly, in the `devbox.json`, should the field be `includes`?

Reply:
"include" is a verb, not a noun so I think it should be singular (e.g. tsconfig.json also uses "include" and not "includes"). @Lagoja curious if you have opinion.

For the range variable, I'll change it, maybe to `included`?

Reply:
oh its a verb! I see i see. 
Okay, we can leave as is for devbox.json, and yeah I like `included` for the range var 👍🏾